cmd/kubeadm/app/apis/kubeadm/v1beta1/types.go

```diff
@@ -34,6 +34,9 @@ type InitConfiguration struct {
 	// InitConfiguration and ClusterConfiguration. Hence, the information must not be duplicated, and is therefore omitted here.
 	ClusterConfiguration `json:"-"`
 
+	// IgnorePreflightErrors holds the list of pre-flight errors to ignore during kubeadm's initialisation.
+	IgnorePreflightErrors []string `json:"ignorePreflightErrors,omitempty"`
```

Comment:
this is already planned in the v1beta2 config, see:
https://github.com/kubernetes/kubeadm/issues/1439

so technically we can accept this PR but only once the v1beta1 types are already created, and we cannot add this to v1beta1. today we are going to have a kubeadm 1.15 planning session at 4PM UTC.
https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle

Reply:
TIL, thanks for pointing to this issue.
I have undone the changes on v1beta1, but now see the below error:
```console
$ make WHAT=cmd/kubeadm
[...]
E0321 15:15:07.858860   27590 conversion.go:667] Warning: could not find nor generate a final Conversion function for k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm.NodeRegistrationOptions -> k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha3.NodeRegistrationOptions
E0321 15:15:07.858900   27590 conversion.go:668]   the following fields need manual conversion:
E0321 15:15:07.858910   27590 conversion.go:670]       - IgnorePreflightErrors
E0321 15:15:07.972950   27590 conversion.go:667] Warning: could not find nor generate a final Conversion function for k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm.NodeRegistrationOptions -> k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta1.NodeRegistrationOptions
E0321 15:15:07.972989   27590 conversion.go:668]   the following fields need manual conversion:
E0321 15:15:07.972996   27590 conversion.go:670]       - IgnorePreflightErrors
[...]
make: *** [all] Error 1
```
...and it isn't obvious where this manual conversion should be added. 
Any pointer?

Reply:
Resolved now that I've rebased on top of latest `master`.